docs/handbook/01_intro.md

```diff
@@ -0,0 +1,20 @@
+# Introduction
+
+Quantum computing has the potential to revolutionize many fields in the 21st century. Over the past decade, numerous quantum computers from multiple providers based on different qubit technologies have been made publicly available.
+However, the best hardware is only as good as the software available to realize corresponding applications on it---a lesson learned from the past decades of research on designing and developing classical circuits and systems.
+Thanks to the software tools and methods for _Electronic Design Automation (EDA)_, we can create classical systems with a staggering amount of transistors and complex functionalities that we often take for granted.
+These methods allow designers to efficiently and automatically handle the intricacies of such systems and optimize their performance.
+Compared to that, most existing software solutions for quantum computing are based on manual approaches.
+This is not only susceptible to errors, inefficiency, and inconsistency but also leaves decades of research on design automation methods underutilized.
+
+The _Munich Quantum Toolkit (MQT)_, which is developed by the [Chair for Design Automation](https://www.cda.cit.tum.de/) at the [Technical University of Munich](https://www.tum.de/), aims to leverage this latent potential by providing a collection of state-of-the-art design automation methods and software tools for quantum computing.
+Our overarching objective is to provide solutions for design tasks across the entire quantum software stack.
+This entails high-level support for end users in realizing their _applications_ as well as efficient methods for the _classical simulation_, _compilation_, and _verification_ of quantum circuits.
+Reaching towards the hardware level, we also consider tools for _quantum error correction_ and _physical design_.
```

Comment:
QEC is never mentioned again in the main text. If nothing is added to the other chapters then this would be a good place to cite our QEC work.

Reply:
Done in https://github.com/cda-tum/mqt/pull/10/commits/0f8a7402fe827d22aadaf7da4975c13205fe4a4c 👍🏼 
In order to stay consistent and not have this look weird, I also added our papers on end-user support and hardware design, as well as all the citations to our papers from the inner sections. I had the feeling it would look rather inconsistent otherwise.

Please let me know, if I missed an important reference here. I tried to cross-check with our webpage.